doc/manual/typing.md

```diff
@@ -190,9 +190,7 @@ The following type constructors are available:
   pair.fst : Num
   ```
 
-- **Dynamic record**: `{_: T}`. A record whose field
-  names are statically unknown but are all of type `T`.  Typically used to model
-  dictionaries.
+- **Dictionary**: `{_: T}`. A dictionary mapping string keys to `T`.
```

Comment:
Nitpick: "[...]to values of type `T`." might be _slightly_ clearer.

Reply:
I think the new version is actually losing some information. There's no notion of "dictionaries" among primitive values (records are the only notion of key-value maps), so if I'm a beginner, I could think that there is something else out there, called dictionary, with a potentially different syntax.

I would suggest to just change

>Dynamic record

to Dictionary and keep the original sentence.